cobalt/media/media_module.cc

```diff
@@ -198,6 +198,24 @@ bool MediaModule::SetConfiguration(const std::string& name, int32 value) {
     LOG(INFO) << (value ? "Enabling" : "Disabling")
               << " media metrics collection.";
     return true;
+  } else if (name == "EnableAudioBufferBudgetOverride") {
```

Comment:
Also, I'd still think that the per stream option (by adding the read/write variable `SourceBuffer.bufferLimit`) is more intuitive for the webapp to use.